polaris-react/src/components/ButtonGroup/ButtonGroup.scss

```diff
@@ -47,7 +48,8 @@
 
     #{$se23} & {
       // stylelint-disable-next-line polaris/z-index/declaration-property-value-allowed-list -- se23 temporary styles
-      z-index: unset;
+      // breaks focus outline for segmented buttons
```

Comment:
Can we remove these rather than comment them out? Same for the commented out code below

Reply:
Yea, definitely, my mistake for missing this clean up